dutch/bos/verlaten-weg/groot-monster/groot-monster

```diff
@@ -0,0 +1,3 @@
+Ai, het was de verkeerde richting. Je stopt abrupt want je ziet 2 grote boze gele ogen naar je kijken.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.